sentry-ruby/spec/isolated/puma_spec.rb

```diff
@@ -4,6 +4,8 @@
 # Because puma doesn't have any dependency, if Rack is not installed the entire test won't work
 return if ENV["RACK_VERSION"] == "0"
 
+SimpleCov.command_name "RSpecIsolated"
```

Comment:
Should we use something like `__FILE__` to make it more explicit (I'm not sure if we'll need to read it though)? But it will save us the headache to name it for future isolated tests.

Reply:
> Note that this only has to be invoked ONCE PER TEST SUITE, so even if you have 200 unit test files, specifying it in some_test.rb is enough.

so even if we add more isolated specs, just having this one here is sufficient